nautobot/virtualization/api/nested_serializers.py

```diff
@@ -67,5 +61,5 @@ class NestedVMInterfaceSerializer(WritableNestedSerializer):
     virtual_machine = NestedVirtualMachineSerializer(read_only=True)
 
     class Meta:
-        model = Interface
+        model = VMInterface
```

Comment:
This bug has existed for years...

Reply:
🙈

Reply:
Now fixed in `develop` via #1705 since this PR won't come in until 1.4.